src/main/java/org/web3j/tx/HighThroughputTransactionManager.java

```diff
@@ -0,0 +1,127 @@
+package org.web3j.tx;
+
+import org.web3j.crypto.Credentials;
+import org.web3j.protocol.Web3j;
+import org.web3j.protocol.core.DefaultBlockParameterName;
+import org.web3j.protocol.core.methods.request.RawTransaction;
+import org.web3j.protocol.core.methods.response.EthBlock;
+import org.web3j.protocol.core.methods.response.EthSendTransaction;
+import org.web3j.protocol.core.methods.response.TransactionReceipt;
+import rx.Subscription;
+
+import java.io.IOException;
+import java.math.BigInteger;
+import java.util.HashMap;
+import java.util.concurrent.ExecutionException;
+
+/**
+ * High Throughput transaction manager.
+ */
+
+public class HighThroughputTransactionManager extends RawTransactionManager {
+
+	//this static hashmap stores web3j objects and their accounts
+	// ---> web3j ---> account ---> HQtransactionManager
+	private static HashMap<Web3j, HashMap<String, HighThroughputTransactionManager>> map = new HashMap<>();
+
+	private String accountAddress;
+	private BigInteger nonce = BigInteger.valueOf(-1);
+	private Web3j web3j;
+	private byte chainId;
+
+	//this private hashmap stores receipts for each block
+	// ---> block number ---> txindex ---> TransactionReceipt
+	private HashMap<BigInteger, HashMap<Long, TransactionReceipt>> transactions;
+
+	public static HighThroughputTransactionManager getInstance(Web3j web3j, Credentials credentials)
+	{
```

Comment:
Please can you ensure code style is consistent with the rest of web3j - { should be on the same line as the method declaration. I will be getting checkstyle rules set up shortly so this will be less of an issue.